tests/detect-lua-01/test.yaml

```diff
@@ -0,0 +1,21 @@
+requires:
+  features:
+    - HAVE_LUA
+
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 2
+    match:
+      event_type: flow
+- filter:
+    count: 1
+    match:
+      event_type: stats
+- filter:
+    count: 2
+    match:
+      event_type: alert
+      alert.signature_id: 6677001
```

Comment:
Considering there are two other rules added to the test, can you add filters for those as well, or, if they're not needed for the test, considering the original ones in detect-lua, you could probably remove them.

Reply:
friendly ping @TheKharleeci

Reply:
Thanks for the friendly ping @catenacyber. I think I'm confused on how to go about this particular test

Reply:
How can we help ?
What about the comments from Juliana ?

Reply:
Currently working on it. Apologies for the late reply